src/locales/de/QueryPage.json

```diff
@@ -11,7 +11,7 @@
     },
     "Buttons": {
         "Submit": "Ergbenisse anzeigen",
```

Comment:
here is a typo

Reply:
I removed the entire entry because it is no longer needed.